source/opt/optimizer.cpp

```diff
@@ -523,6 +523,10 @@ bool Optimizer::RegisterPassFromFlag(const std::string& flag) {
     RegisterPass(CreateInterpolateFixupPass());
   } else if (pass_name == "remove-dont-inline") {
     RegisterPass(CreateRemoveDontInlinePass());
+  } else if (pass_name == "analyze-dead-output-stores") {
+    RegisterPass(CreateAnalyzeDeadOutputStoresPass(&live_locs_));
+  } else if (pass_name == "eliminate-dead-output-stores") {
+    RegisterPass(CreateEliminateDeadOutputStoresPass(&live_locs_));
```

Comment:
Making this available through the command line for users is not part of this PR. For the purposes of this PR, I am only making this available through the API. These command options above are just for debugging purposes. They are not advertised to users. I will add comments here so that is better understood by everyone.

Reply:
If the CLI is not suppose to be part of this PR, then just remove these.  Even if the option is not mentioned on the help screen, people can just look at the source to see the options.